src/pal/src/configure.cmake

```diff
@@ -933,6 +933,8 @@ if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
   set(PAL_PT_READ_D PT_READ_D)
   set(PAL_PT_WRITE_D PT_WRITE_D)
   set(HAS_FTRUNCATE_LENGTH_ISSUE 1)
+  set(HAVE_SCHED_OTHER_ASSIGNABLE 1)
```

Comment:
Even if we could play with cache - this approach is wrong. `SCHED_OTHER` is by the POSIX standard defined as implementation-specific, generalizing isn't valid.